deps.ffmpeg/60-mbedtls.zsh

```diff
@@ -3,21 +3,21 @@ autoload -Uz log_debug log_error log_info log_status log_output
 ## Dependency Information
 local name='mbedtls'
 local -A versions=(
-  macos 3.2.1
-  linux 3.2.1
-  windows 3.2.1
+  macos 3.4.0
+  linux 3.4.0
+  windows 3.4.0
 )
 local url='https://github.com/Mbed-TLS/mbedtls.git'
 local -A hashes=(
-  macos 869298bffeea13b205343361b7a7daf2b210e33d
-  linux 869298bffeea13b205343361b7a7daf2b210e33d
-  windows 869298bffeea13b205343361b7a7daf2b210e33d
+  macos 1873d3bfc2da771672bd8e7e8f41f57e0af77f33
+  linux 1873d3bfc2da771672bd8e7e8f41f57e0af77f33
+  windows 1873d3bfc2da771672bd8e7e8f41f57e0af77f33
 )
 local -a patches=(
   "macos ${0:a:h}/patches/mbedtls/0001-enable-posix-threading-support.patch \
     ea52cf47ca01211cbadf03c0493986e8d4e0d1e9ab4aaa42365b2dea7b591188"
-  "linux ${0:a:h}/patches/mbedtls/0001-enable-posix-threading-support.patch \
-    ea52cf47ca01211cbadf03c0493986e8d4e0d1e9ab4aaa42365b2dea7b591188"
+  "* ${0:a:h}/patches/mbedtls/0002-enable-dtls-srtp-support.patch \
```

Comment:
You seem to have removed the patch enabling Posix threads on Linux - please leave this in and just add the new patch.

Reply:
@PatTheMav fixed! Sorry about that

Reply:
This appears to have been fixed. @PatTheMav